charts/mysqldump/templates/configmap-script.yaml

```diff
@@ -85,6 +85,13 @@ data:
     rcu=$?
     {{ end }}
 
+    {{ if .Values.upload.s3.enabled -}}
+    echo "upload files to s3 storage bucket {{ .Values.upload.s3.bucketname }}"
+    aws configure set aws_access_key_id "$S3_ACCESSKEY" && aws configure set aws_secret_access_key "$S3_SECRET_KEY" && aws configure set region "$S3_REGION"
```

Comment:
cant we use a credentials secret here, as for the other methods?

Reply:
If credentials are not used, the only way to upload with temporary credentials is through AWS STS.